### PR TITLE
Initial build PR

### PR DESCRIPTION
**Pull request by varas**
Build containerised agent "bundle" with simple make command. See attached README.

### CODE REVIEW

File: build/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,27 @@
+ARG agent_version=latest
+ARG base_image=newrelic/infrastructure:${agent_version}
+
```
Comment (mariomac) [inline build/Dockerfile]: i would create another `ARG` for `openjdk8-jre` version.

File: build/nri-integrations [hunk 2]
```diff
@@ -0,0 +1,25 @@
+#name,packaging,version,arch(optional, amd64 is the default)
+nri-docker,base,1.1.1
+nri-flex,base,0.9.3
```
Comment (davidbrota) [inline build/nri-integrations]: I think we should remove docker and flex from this file

Comment (varas) [reply build/nri-integrations]: Indeed, good one! I'll document these are being bundled within the base image

File: build/build.sh [hunk 3]
```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env bash
+
```
Comment (carlosroman) [inline build/build.sh]: I don't think this file is needed as this was used for the base image.

File: README.md [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+1. Get your desired containerised Infrastructure agent from the tags avaiable at
+[`newrelic/infrastructure`](https://hub.docker.com/r/newrelic/infrastructure/tags)
+
+1. NR will keep latest stable integrations versions at `build/nri-integrations`.
+   > You could potentially edit the file and set your desired ones at your own risk.
+
+1. Run the following command:
+
+   ```bash
+   make VERSION=<bundle version> AGENT_VER=<version of agent>
```
Comment (davidbrota) [inline README.md]: AGENT_VER**SION**

File: build/nri-integrations (hunk 2 above)
Comment (varas) [reply build/nri-integrations]: I'm removing the bundle column from the file as well

File: build/build.sh (hunk 3 above)
Comment (carlosroman) [reply build/build.sh]: Oh I see. You've taken the old script and used that to build this image. Ignore me :)

Comment (varas) [reply build/build.sh]: it's the actual bundle build script

**Pull request merged**